--- ifplugd-0.26/configure.ac.orig	2005-05-02 17:57:57.000000000 +0100
+++ ifplugd-0.26/configure.ac	2005-05-02 17:58:23.000000000 +0100
@@ -45,7 +45,7 @@
    CFLAGS="$CFLAGS -pipe -W -Wall -pedantic"
 
    AC_LANG_CONFTEST([int main() {}])
-   $CC -c conftest.c -std=c99 -Wno-unused-parameter $CFLAGS > /dev/null 2> /dev/null && CFLAGS="$CFLAGS -std=c99 -Wno-unused-parameter"
+   $CC -c conftest.c -Wno-unused-parameter $CFLAGS > /dev/null 2> /dev/null && CFLAGS="$CFLAGS -Wno-unused-parameter"
    rm -f conftest.o
 fi
 
